### PR TITLE
Add back arrows on child screens

### PR DESCRIPTION
**Pull request by alorma**
screenshot_1526393060

### CODE REVIEW

File: app/src/main/java/io/stanwood/bitrise/ui/build/ui/BuildFragment.kt [hunk 1]
```diff
@@ -46,4 +51,14 @@ class BuildFragment : Fragment() {
                 lifecycle.addObserver(viewModel)
                 vm = viewModel
             }.root
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        view.findViewById<Toolbar>(R.id.toolbar)?.let {
```
[the remark this answers is not in the document]
Comment (alorma) [reply app/src/main/java/io/stanwood/bitrise/ui/build/ui/BuildFragment.kt]: Oh i copied the findView toolbar from somewhere

You want to go back, not exit the app (or does exit just close current?)

[the remark this answers is not in the document]
Comment (ubuntudroid) [reply app/src/main/java/io/stanwood/bitrise/ui/build/ui/BuildFragment.kt]: `exit()` just goes back in the backstack like a press on the system back button would. I agree that it's a bit unfortunately named. ;)

**Comment by alorma** [comment]
Oh, i will apply it then

**Comment by alorma** [comment]
Updated @ubuntudroid

**Comment by ubuntudroid** [comment]
Can you also change the `findViewById()` and set the up navigation button from XML like suggested? 🙏

**Comment by alorma** [comment]
Done

**Comment by ubuntudroid** [comment]
@mmoczkowski Please review as well and merge if you're happy.

**Comment by ubuntudroid** [comment]
@mmoczkowski Just a kind reminder to review and merge this one :)

**Pull request closed without merging**